File: xiajiwangluo_store/src/main/java/com/xiajiwangluo/service/TotalService.java
package com.xiajiwangluo.service;

import com.xiajiwangluo.bean.Inout;
import com.xiajiwangluo.bean.Total;

import java.util.List;

public interface TotalService {

    /**
     * 查询销售统计
     * */
    List<Total> findAll(String sid,String day);
    /**
     * 查询退货统计
     * */
    List<Inout> findDrawback(String sid, String day);

}

File: xiajiwangluo_store/src/main/webapp/js/service/totalService.js
//服务层
app.service('totalService',function($http){


    //查询销售统计
    this.findAll=function(day){
        return $http.get('../total/findAll.do?day='+day);
    };

    //查询退货统计
    this.findDrawback=function(day){
        return $http.get('../total/drawback.do?day='+day);
    };

    //导出统计
    this.exportTotalAll=function(totalList){
        return $http.post('../export/total.do',totalList);
    };

    //导出退货
    this.exportDrawback=function(inoutList){
        return $http.post('../export/drawback.do',inoutList);
    };


});